sdk/javascript/generator/StructTypeFormatter.py

```diff
@@ -248,9 +255,11 @@ def generate_deserialize_field(self, field, aligned_descriptor, arg_buffer_name=
 		if not use_custom_buffer_name:
 			buffer_load_name = 'view.buffer'
 
-		load_aligned = self.struct.is_aligned if not aligned_descriptor else aligned_descriptor['generate_aligned']
+		use_aligned_deserializer = self.struct.is_aligned
+		if DeserializerMode.USE_DEFAULT != deserializer_mode:
+			use_aligned_deserializer = True if DeserializerMode.USE_ALIGNED == deserializer_mode else False
```

Comment:
should reduce to `DeserializerMode.USE_ALIGNED == deserializer_mode` ?

Reply:
nope, that's the whole point....